main/zoe/api/records.md

```diff
@@ -0,0 +1,151 @@
+# Records
+
+<Zoe-Version/>
+
+These records (JavaScript objects with data properties and no
+functions) are used throughout Zoe.
+
+## SeatAndPayout
+
+This is returned by a call to `redeem` on Zoe. A seat is an arbitrary object whose methods allow the user to take certain actions in a contract. The payout is a promise that resolves to an object which has keywords as keys and promises for payments as values. Note that while the payout promise resolves when an offer is completed, the promise for each payment resolves after the remote issuer successfully withdraws the payment.
+
+```js
+someSeatAndPayout: {
```

Comment:
Using property-colon notation both on the outside, where the name is only descriptive, and on the inside, where usually is actually a literal name you might use, is confusing. It is too confusing where you use `someExitRule:` below, because the only property name that would actually appear there under normal use, and under the only use you explain, is `exit:`